packages/waku/src/cli.ts

```diff
@@ -155,6 +159,7 @@ Options:
   --with-vercel         Output for Vercel on build
   --with-cloudflare     Output for Cloudflare on build
   --with-deno           Output for Deno on build
+  --with-aws-lambda     Output for AWS Lambda on build
```

Comment:
Just wondering if `--with-lambda` might work or not. It's too unclear maybe.

Reply:
On second thought, I prefer `--with-aws-lambda`.

Reply:
I agree - this is more clear